swan-lake/development-tutorials/run-in-the-cloud/code-to-cloud-deployment.md

```diff
@@ -21,6 +21,8 @@ To complete this tutorial, you need:
 1. <a href="https://www.docker.com/" target="_blank">Docker</a> installed and configured in your machine
 2. <a href="https://kubernetes.io/docs/tasks/tools/" target="_blank">Kubectl</a> installed and configured in a <a href="https://minikube.sigs.k8s.io/docs/start/" target="_blank">Kubernetes cluster</a>
 
+>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.
```

Comment:
Instead of saying 

```
run `export DOCKER_DEFAULT_PLATFORM=linux/amd64`
```

should we say something like

> set an environment variable `DOCKER_DEFAULT_PLATFORM` to `linux/amd64`.

Reply:
Since this is valid only for macOS, the command should do IMO.

Reply:
I think it'll still be good to explicitly say set an environment variable. We can give the snippet too.

Reply:
Fixed